### DISCUSSION TITLE
Linking a repository to a project description

### DISCUSSION THREAD

**Discussion by MohamedIbrahimZineldin** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#70] [Q&A]
Hey there,
how can I link my project's GitHub repository to the project description page and share it externally?
I new this step is so important but I tried many times and I can't find what this page says : https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/linking-a-repository-to-a-project-board
May Anyone help me?

**Comment by robertogl** [comment]
Hi @MohamedIbrahimZineldin 

Do not worry about that, I will do it using GitHub submodules. Sorry for any confusion.

**Reply by MohamedIbrahimZineldin** [reply]
Thank you very much , All people here are very kind.

**Reply by robertogl** [reply]
:) thanks!
